scripts/helpers/tizen_gdbserver_run.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+SDB="$TIZEN_SDK_ROOT/tools/sdb"
+PORT=9999
+RESULT_MODE="debug"
+
+function help() {
+    cat <<EOF
```

Comment:
Why not a good old `echo` ? Also, I would use "$0" without basename, unless this script is on the PATH.

Reply:
I've forgotten to add a description to arguments, and I think that multiline help it look cleaner.